### PR TITLE
Add path tests to Karma

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1305
[Problem] Path tests not included
[Solution] Include path tests to Karma

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
when reviewing please use option
image

**Comment by lslachciak** [comment]
%lines before 57.1
%lines after 57.79

File: tests/js/core/util/path/path.js [hunk 1]
```diff
@@ -1,179 +1,204 @@
...
+		});
+	}
+
+	if (typeof define === "function") {
+		define(function () {
+			return runTests;
+		});
+	} else {
+		runTests(tau.util.path, window.helpers,
+			tau);
```
Comment (DohyungLim) [inline tests/js/core/util/path/path.js]: line remove, and attach previous line :)

File: tests/js/core/util/path/path.js [hunk 2]
```diff
@@ -1,179 +1,204 @@
...
+			parent.innerHTML = HTML;
+		}
+
+		module("core/util/path", {
+			setup: initHTML
+		});
+
+		test("getDocumentUrl", function () {
+			var urlObject = {
+
```
Comment (DohyungLim) [inline tests/js/core/util/path/path.js]: delete line. and attach below line to upper line. I think merge line that looks good.

**Review by DohyungLim — COMMENTED** [review]
2 comment.

**Review by korneliakobiela — APPROVED** [review]
LGTM

File: tests/js/core/util/path/path.js (hunk 1 above)
Comment (lslachciak) [reply tests/js/core/util/path/path.js]: done

File: tests/js/core/util/path/path.js (hunk 2 above)
Comment (lslachciak) [reply tests/js/core/util/path/path.js]: done

**Pull request merged**